homeassistant/components/stookalert/binary_sensor.py

```diff
@@ -0,0 +1,88 @@
+"""This component provides support for Stookalert Binary Sensor."""
+from datetime import timedelta
+import logging
+
+import stookalert
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA, BinarySensorDevice
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME
+from homeassistant.helpers import config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+CONF_PROVINCE = "province"
+DEFAULT_NAME = "Stookalert"
+ATTRIBUTION = "Data provided by rivm.nl"
+PROVINCES = [
+    "Drenthe",
+    "Flevoland",
+    "Friesland",
+    "Gelderland",
+    "Groningen",
+    "Limburg",
+    "Noord-Brabant",
+    "Noord-Holland",
+    "Overijssel",
+    "Utrecht",
+    "Zeeland",
+    "Zuid-Holland",
+]
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_PROVINCE): vol.In(PROVINCES),
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    }
+)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Stookalert binary sensor platform."""
+    province = config.get(CONF_PROVINCE)
```

Comment:
Please use `dict[key]` for required keys and keys with default config schema values.
```suggestion
    province = config[CONF_PROVINCE]
```